src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/RandomNumberGenerator.cs

```diff
@@ -165,6 +166,200 @@ public static byte[] GetBytes(int count)
             return ret;
         }
 
+        /// <summary>
+        ///   Fills the elements of a specified span with items chosen at random from choices.
+        /// </summary>
+        /// <param name="choices">The items to use to fill the buffer.</param>
+        /// <param name="destination">The buffer to receive the items.</param>
+        /// <typeparam name="T">The type of items.</typeparam>
+        /// <exception cref="ArgumentException">
+        ///   <paramref name="choices" /> is empty.
+        /// </exception>
+        /// <seealso cref="GetString" />
+        /// <seealso cref="GetHexString(Span{char}, bool)" />
+        public static void GetItems<T>(ReadOnlySpan<T> choices, Span<T> destination)
+        {
+            if (choices.IsEmpty)
+                throw new ArgumentException(SR.Arg_EmptySpan, nameof(choices));
+
+            GetItemsCore<T>(choices, destination);
+        }
+
+        /// <summary>
+        ///   Creates an array populated with items chosen at random from choices.
```

Comment:
Where does the leading two spaces come from in these -- VS tooling? We seem to not generally do that. I'm guessing tooling trims these on the display side though.

Reply:
> Where does the leading two spaces come from in these -- VS tooling?

I use VS Code without any extensions. I just typed them that way.

> I'm guessing tooling trims these on the display side though.

Right. It collapses white space similarly to how XML does it.

I can get rid of the indentation if we don't like it.

Reply:
As far as I know, it comes from me.  We (and, indeed, most humans) indent XML in files, but for some reason xmldoc is generally not indented.  That makes it hard to read (text aligning with structure).  So I apply normal human+XML rules, indenting the content of a tag.

As the /// comment just turns into xml output from the compiler, XML's notion of relevant vs irrelevant whitespace applies.

Reply:
> That makes it hard to read

FWIW, I personally find the extra indentation jarring.  Likely just because most xml doc comments I see don't have it, I don't write them myself, etc.